samples/sdl/main.c

```diff
@@ -187,7 +187,7 @@ static void
 quit(int rc)
 {
     SDL_VideoQuit();
-    _exit(rc);
+    Reboot(rc);
```

Comment:
There's only 1 single instance where SDL video isn't initialized yet.
Consider merging `quit` and `Reboot` into one function - they do almost exactly the same.

You could just move `SDL_VideoQuit` into the caller for example.

Reply:
I still think this argument is valid; especially as we use arbitrary return codes (1 and 5; whatever those mean).

Eitherway, this is "better than master" and it should be merged soon.